integreat_cms/cms/templates/feedback/admin_feedback_list_archived.html

```diff
@@ -0,0 +1,166 @@
+{% extends "_base.html" %}
+{% load i18n %}
+{% load static %}
+{% load widget_tweaks %}
+{% load text_filters %}
+{% block content %}
+    {% with filter_form.filters_visible as filters_visible %}
+        <div class="table-header">
+            <div class="flex flex-wrap justify-between">
+                <h1 class="heading">{% translate "Archived feedbacks" %}</h1>
```

Comment:
```suggestion
                <h1 class="heading">{% translate "Archived feedback" %}</h1>
```
This is picky, but as far as I could research the plural of feedback is feedback

Reply:
Just a matter of curiosity: if I say in German “die Feedbacks”, plural has “s”, right?